### ISSUE TITLE
Fonts not working in android 11

### ISSUE THREAD

**Issue by devnilspatel** [y0av/MaterialClockWidget#2]
Its working fine before, but after software update custom fonts are not working. Its only showing default fonts in widget.

**Comment by y0av** [comment]
> widget

Yeah. I tried solving it but i think it might be impossible to do in android 11 and 12.

**Comment by Awooiel** [comment]
Please fix this would be nice <3

**Comment by y0av** [comment]
Unfortunately Google has removed the ability to use custom fonts in app widgets and they don't intend to allow it as stated here: https://issuetracker.google.com/issues/190444596

**Comment by Awooiel** [comment]
That's unfortunate.
Thank you for the work though
